src/io/flutter/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -552,8 +553,8 @@ public void received(Isolate isolate) {
                 else if (eventKind == EventKind.Resume) {
                   // Currently true if we got here via 'flutter attach'
                   ApplicationManager.getApplication().invokeLater(() -> {
-                    getSession().initBreakpoints();
-                    myVmServiceWrapper.attachIsolate(isolateRef);
+                    myVmServiceWrapper.attachIsolate(isolateRef, isolate);
+                    // TODO(messick) onVmConnected(vmService) here, to connect inspector? Seems to work without, not sure why.
```

Comment:
The inspector is smart enough to generally be fine even if the isolate has already started because the inspector window might be opened after the app is already started. Until the inspector window is actually opened we avoid doing work to keep the inspector from degrading app performance.

Reply:
I deleted that TODO.